syntax = "proto3";

package Hello;


service Greeter{
    rpc SayHello(HelloRequest) returns (HelloReply);
    rpc LotsOfReplies(HelloRequest) returns (stream HelloReply);
    rpc LotsOfGreetings(stream HelloRequest) returns(HelloReply);
}

message HelloRequest{
    string name = 1;
}

message HelloReply{
    string reply = 1;
}